shared/modules/network.utils.ts

```diff
@@ -35,7 +33,7 @@ export function isPrefixedFormattedHexString(value) {
  * @param chainId - ChainID of network
  * @returns Whether the current network supports token detection
  */
-export function isTokenDetectionEnabledForNetwork(chainId) {
+export function isTokenDetectionEnabledForNetwork(chainId: string | undefined) {
```

Comment:
Initially this was `unknown`, but I decided to use `string | undefined` based on how this function is used in practice.